frontend/components/hypercert-create.tsx

```diff
@@ -516,7 +522,18 @@ const exportAsImage = async (id: string) => {
     //useCORS: true,
     proxy: "https://cors-proxy.hypercerts.workers.dev/",
     imageTimeout: 0,
+  }).catch((e) => {
```

Comment:
Are we sure that html2canvas will throw an error if fetching the image fails? Worth some testing either manually or via unit/integration tests.

Reply:
Actually now that I think about it, I don't think this solves the issue, because without the catch any thrown errors should stop everything. Instead the behavior we see is that the hypercert mints anyway with the broken images.

So catching is insufficient....

Reply:
Yep found the same during testing rn. I'm going to think about an alternative route. Because it's not like the canvas will be empty (that'd be easy to check), it's missing one specific layer